solutions/search/query-rules-ui.md

```diff
@@ -0,0 +1,136 @@
+---
+applies_to:
+  stack: ga 9.1
+  serverless: ga
+  elasticsearch:
+---
+
+# Query rules UI
+Use query rules to boost, pin, or exclude specific documents when queries contain certain keywords, phrases, or match defined search patterns.
```

Comment:
This is incorrect, we only support pinning or excluding. When I read boost I don't think of pinning documents, I think of increasing the likelihood that they show up in the results, and unfortunately this is not supported. You could consider linking here to https://www.elastic.co/docs/reference/elasticsearch/rest-apis/searching-with-query-rules for more information about query rules as well. (I know you link to it further down)